lib/qesapdeployment.pm

```diff
@@ -413,4 +414,59 @@ sub qesap_ansible_cmd {
     assert_script_run($ansible_cmd);
 }
 
+
+=head3 qesap_ansible_script_output
+
+    Use Ansible to run a command remotely and get the stdout.
+    Command is executed with elevated privileges
+
+    qesap_ansible_script_output(cmd => 'crm status', provider => 'aws', host => 'vmhana01');
+
+    It uses playbook data/sles4sap/script_output.yaml
+
+    1. ansible-playbook run the playbook, remote cmd ar paramenter
+    2. the playbook executes the command and redirects the output to file, both remotely
+    3. the playbook download the file locally
+    4. the file is read and sored to be returned to the caller
+
+=over 3
+
+=item B<PROVIDER> - Cloud provider name, used to find the inventory
+
+=item B<CMD> - command to run remotely
+
+=item B<HOST> - filter hosts in the inventory
+
+=back
+=cut
+
+sub qesap_ansible_script_output {
+    my (%args) = @_;
+    croak 'Missing mandatory cmd argument' unless $args{cmd};
+    croak 'Missing mandatory host argument' unless $args{host};
+    my $inventory = qesap_get_inventory($args{provider});
+
+    if (script_run('test -e script_output.yaml') != 0) {
+        my $cmd = join(' ',
+            'curl', '-v', '-fL',
+            data_url('sles4sap/script_output.yaml'),
+            '-o', 'script_output.yaml');
+        assert_script_run($cmd);
+    }
+
+    # ansible-playbook -l vmhana01 -vvvv -i ~/p_repo/qe-sap-deployment/terraform/azure/inventory.yaml script_output.yaml -e "cmd='crm status'"
+    my $ansible_cmd = join(' ',
+        'ansible-playbook',
+        '-l', $args{host},
+        '-vvvv',
+        '-i', $inventory,
+        'script_output.yaml',
+        '-e', "\"cmd='$args{cmd}'\"");
+    script_run('rm /tmp/ansible_script_output/testout.txt || echo "Nothing to clean"');
```

Comment:
Should this be a `script_run` if you don't care about its return value?

I'd go simply with:
```
enter_cmd 'rm -f /tmp/ansible_script_output/testout.txt';
```

Also, same suggestion regarding `/tmp/ansible_script_output/testout.txt`: you use it already 3 times in the function. You can define it in a variable at the start of the function and then if you ever need to change it, it would be enough to change it in the one place.